README.md

```diff
@@ -79,7 +79,10 @@ If the system does not detect any JDK, the extension will offer a downloader and
 Alternatively, you can manually specify the path to JDK binaries by utilizing the JDK downloader.
 You can also access the JDK downloader through the "Download, install, and Use JDK" option in the command palette.  
 ![JDK Downloader](vscode/images/jdk_downloader.png) 
+### Enabling Java Preview Features 
```

Comment:
Minor nit: This section should perhaps be 1 heading level up, and not under the *JDK Downloader* section; i.e. `## Enabling Java Preview Features`

Reply:
Sure, makes sense.